### DISCUSSION TITLE
Get failed test name from xcresult report

### DISCUSSION THREAD

**Discussion by alexnot95** [lyndsey-ferguson/fastlane-plugin-test_center#343] [General]
Hi @lyndsey-ferguson is it possible to get test  name which is failed? For example I have 2 tests that failed in xcresult report, can I somehow get test names? I know that I can get failed tests  quantity, but what about names? thanks

**Comment by lyndsey-ferguson** [comment]
Yes, indeed it is possible to get the tests that failed. Use the [`tests_from_xcresult`](https://github.com/lyndsey-ferguson/fastlane-plugin-test_center/blob/master/docs/feature_details/tests_from_xcresult.md) action to get the `:failed` tests. Does that help?

**Reply by alexnot95** [reply]
hmm, I'm using this action, but does it return a name? cuz for now, I just used to count failed tests, ot it's return an array of failed test names?

**Reply by lyndsey-ferguson** [reply]
It returns an array of failed test names. I'll update the docs to reflect that.

**Reply by alexnot95** [reply]
thank you

**Reply by lyndsey-ferguson** [reply]
I just [updated the docs](https://github.com/lyndsey-ferguson/fastlane-plugin-test_center/commit/b0d8117c14be46a774d80d340dd49ce98733bebf)
